PrestaShop/PrestaShop
# [*] CORE : Update swift mailer

## Description



Guess it's 'bout time, init?
- This PR updates updates the version of Swift to `5.2.2` and makes the class `Mail` compatible with the newer version of swift.
- Improves some of the PHPDocs
- Provides backward compatibility as swift required SMTP encryption to be set to `'off'` in order to disable encryption, but should now be `false` instead
## Steps to Test this Fix


- Send test emails
- Order something in the store.
- Try with sendmail and with a direct SMTP connection.
- Gmail, Outlook, etc. (STARTTLS) are supported with this update. It should be possible to relay emails to these email providers.
